src/components/agenda-page/DayOfWeekList.tsx

```diff
@@ -73,30 +68,36 @@ const DayOfWeekList: React.FC<DayOfWeekListProps> = ({ dayOfWeek, uid }) => {
             disabled={addingNewTask}
             onClick={!addingNewTask ? () => setAddingNewTask(true) : undefined}
           />
-          <IconButton
-            icon={editingDay ? faChevronUp : faChevronDown}
-            onClick={() => setEditingDay(!editingDay)}
-            size="sm"
-          />
         </div>
       </div>
-      <div className="flex flex-col gap-1 mt-1 p-1">
-        {dayTasks.map((t) => (
-          <TaskItem task={t} uid={uid} key={t.taskID} canEdit={editingDay} />
-        ))}
-        {addingNewTask ? (
-          <NewTaskForm
-            setAddingNewTask={setAddingNewTask}
-            dayOfWeek={dayOfWeek}
-            uid={uid}
-          />
-        ) : (
-          <NewTaskPrompt
-            className="group-hover:visible invisible group-hover:animate-in group-hover:duration-300 group-hover:fade-in"
-            setAddingNewTask={setAddingNewTask}
-          />
+      <Droppable droppableId={getWeekdayName(dayOfWeek)} type="COLUMN">
+        {(droppableProvided: DroppableProvided) => (
+          <div
+            ref={droppableProvided.innerRef}
+            {...droppableProvided.droppableProps}
+          >
+            <div className="flex flex-col mt-2">
+            {dayOfWeekTasks.map((t, index) => (
+              <TaskItem task={t} index={index} uid={uid} key={t.taskID} />
+            ))}
+            </div>
+
+            {addingNewTask ? (
+              <NewTaskForm
+                setAddingNewTask={setAddingNewTask}
+                dayOfWeek={dayOfWeek}
+                uid={uid}
```

Comment:
I feel like passing uid around is a lil sussy, maybe having a global const or something makes more sense

Reply:
I could put it into state but I'd need to null check at every use, just more lines of code. Will probably do this at some point anyways. For now, using it as prop is fine; no security issues